enterprise_access/apps/content_assignments/utils.py

```diff
@@ -15,3 +17,15 @@ def chunks(a_list, chunk_size):
 def format_traceback(exception):
     trace = ''.join(traceback.format_tb(exception.__traceback__))
     return f'{exception}\n{trace}'
+
+
+def are_dates_matching_with_day_offset(days_offset, target_date, date_to_offset):
```

Comment:
Just a suggestion, you could change the name/signature of this function to clarify its intent a little.  The question you're trying to answer here is like "is this date 30 days from today?"  The function could be more precise in trying to answer that question, like
```python
def is_date_n_days_from_now(target_datetime, num_days):
    future_datetime = timezone.now() + timezone.timedelta(days=num_days)
    return target_datetime.date() == future_datetime.date()
```
Also, this function might be better off (for now) being in that `content_metadata_api.py` file.